expression/builtin_like_test.go

```diff
@@ -59,26 +59,31 @@ func (s *testEvaluatorSerialSuites) TestCILike(c *C) {
 	collate.SetNewCollationEnabledForTest(true)
 	defer collate.SetNewCollationEnabledForTest(false)
 	tests := []struct {
-		input   string
-		pattern string
-		match   int
+		input        string
+		pattern      string
+		generalMatch int
+		unicodeMatch int
 	}{
-		{"a", "", 0},
-		{"a", "a", 1},
-		{"a", "á", 1},
-		{"a", "b", 0},
-		{"aA", "Aa", 1},
-		{"áAb", `Aa%`, 1},
-		{"áAb", `%ab%`, 1},
-		{"áAb", `%ab`, 1},
-		{"ÀAb", "aA_", 1},
-		{"áééá", "a_%a", 1},
-		{"áééá", "a%_a", 1},
-		{"áéá", "a_%a", 1},
-		{"áéá", "a%_a", 1},
-		{"áá", "a_%a", 0},
-		{"áá", "a%_a", 0},
-		{"áééáííí", "a_%a%", 1},
+		// general_ci and unicode_ci same
+		{"a", "", 0, 0},
+		{"a", "a", 1, 1},
+		{"a", "á", 1, 1},
+		{"a", "b", 0, 0},
+		{"aA", "Aa", 1, 1},
+		{"áAb", `Aa%`, 1, 1},
+		{"áAb", `%ab%`, 1, 1},
+		{"áAb", `%ab`, 1, 1},
+		{"ÀAb", "aA_", 1, 1},
+		{"áééá", "a_%a", 1, 1},
+		{"áééá", "a%_a", 1, 1},
+		{"áéá", "a_%a", 1, 1},
+		{"áéá", "a%_a", 1, 1},
+		{"áá", "a_%a", 0, 0},
+		{"áá", "a%_a", 0, 0},
+		{"áééáííí", "a_%a%", 1, 1},
```

Comment:
Please add the case for `"ß"` and `"ss"`(and some other pairs like `ss`, `%ß%` maybe), and explain the behavior in the comments:

> performs matching on a per-character basis
https://dev.mysql.com/doc/refman/5.7/en/string-comparison-functions.html#operator_like